conda/environments/cuml_dev_cuda10.1.yml

```diff
@@ -0,0 +1,27 @@
+name: cuml_dev
+channels:
+- rapidsai
+- nvidia
+- rapidsai-nightly
+- conda-forge
+dependencies:
+- cudatoolkit=10.1
+- libclang=8.0.0
+- cmake=3.14.5
+- numba=0.45*
+- rapidsai/label/cuda10.1::cupy>=6*
+- cudf=0.10*
+- rmm=0.10*
+- cython>=0.29,<0.30
+- pytest>=4.6
+- scikit-learn>=0.21
+- umap-learn>=0.3.9
+- scikit-learn>=0.21
+- dask=2.3.0
+- distributed=2.3.0
+- dask-ml
+- dask-cuda=0.9*
```

Comment:
why is dask-cuda still at 0.9?

Reply:
Oversight, will correct it in a follow pr for the same reason I mentioned in your first comment